generate/generate.go

```diff
@@ -13,9 +13,12 @@ import (
 	"strings"
 
 	"github.com/GannettDigital/jstransform/jsonschema"
+	"github.com/tinylib/msgp/gen"
+	"github.com/tinylib/msgp/parse"
+	"github.com/tinylib/msgp/printer"
 )
 
-const disclaimer = "// This file was autogenerated by https://github.com/GannettDigital/jstransform\n\n"
+const disclaimer = "// Code generated by github.com/GannettDigital/jstransform; DO NOT EDIT."
```

Comment:
aren't the newlines necessary?

Reply:
Deven took them out of the string and put them into the format command where they are used.

Reply:
Yes, since I was changing the string anyhow (to match current Go standards for generated code), I thought it made more sense to have the newlines in the format string instead of this constant string.